package.json

```diff
@@ -30,7 +30,8 @@
   "dependencies": {
     "babel-runtime": "5.8.24",
     "fbjs": "^0.7.0",
-    "react-static-container": "^1.0.0-alpha.1"
+    "react-static-container": "^1.0.0-alpha.1",
+    "babel-relay-plugin": "0.7.1"
```

Comment:
Npm3 will throw an error if it isn't listed under 'dependencies', that's why I put it there